Samples/WindowsStudio/README.md

```diff
@@ -1,17 +1,48 @@
+
+## WSE v2 effects
+It also taps into newer effects available in version 2 that are exposed using a set of DDIs custom to Windows Studio Effects. Since these are exclusive to Windows Studio Effects and shipped outside the OS, their defninition is not part of the Windows SDK and has to be copied into your code base ([see DDI documentation](<./Windows Studio Effects DDIs.md>)):
+- Portrait Light (*[DDI documentation](<./Windows Studio Effects DDIs.md#ksproperty_cameracontrol_windowsstudio_stagelight-control>)*)
+- Creative Filters (Animated, Watercolor and Illustrated) (*[DDI documentation](<./Windows Studio Effects DDIs.md#ksproperty_cameracontrol_windowsstudio_creativefilter-control>)*)
+
+## WSE limits frame formats and profiles
+The code sample allows to see all the MediaTypes (frame formats) exposed by the camera and change the [camera profile](https://learn.microsoft.com/en-us/windows/uwp/audio-video-camera/camera-profiles) it is provisioned with. When initialized with profile for processing effects, *WSE* limits:
+- The number of stream available from a source with effects applied to 1
+- Processing of only color stream (i.e. no infrared stream, etc.) 
+- MediaTypes exposed with:
+  - Resolutions of at most 2560x1440 and at least 640x360
+  - Framerate of at most 30fps and at least 15fps
+  - Subtype in NV12 format only (*WSE* may take as input other subtypes, but will currently only expose out NV12)
+
+That said as alluded to above, these limits are only imposed when the camera profile used is one of the below [known video profile](https://learn.microsoft.com/en-us/uwp/api/windows.media.capture.knownvideoprofile?view=winrt-26100):
+- the default (or not specified), also known as *Legacy* profile
+- *VideoRecording*
+- *VideoConferencing*
+
+**With other profiles however, while these limits are not imposed and instead rely on the capabilities defined by the original camera driver, none of the Windows Studio Effects DDIs are supported**. This is why for example, the Windows Camera Application might not support any effects when entering photo capture mode but offer a different set of MediaTypes to exercise such as with higher resolution and other subtypes.
+
+*WSE* also always defines and exposes a custom *"passthrough"* profile that exposes all MediaTypes available (*[see DDI doc](<./Windows Studio Effects DDIs.md#WSE-custom-profile>)*). This is meant to enable application scenarios such as initializing the camera to record video at higher resolution than 1440p or higher framerate than 30fps when the camera supports these capabilities but is limited by *WSE* in order to apply effects.
```

Comment:
>custom *"passthrough"* profile [](http://example.com/codeflow?start=40&length=30)

About custom passthrough profile. I didn't think the original WSE v1 systems had this. Did we have this available from first shipped WSE enabled system onwards?

Reply:
true some older systems (8280 and KMB) do not have this profile supported., PHX+, MTL+, 8380 do have it, will rephrase